ENV Variables missing from remote.yaml for the pfcon deployment

**error: log**

```
kubectl logs pfcon-5f955ddb7c-x8vm6
> [2022-04-11 16:21:58 +0000] [1] [INFO] Starting gunicorn 20.1.0
> [2022-04-11 16:21:58 +0000] [1] [INFO] Listening at: http://0.0.0.0:30005 (1)
> [2022-04-11 16:21:58 +0000] [1] [INFO] Using worker: sync
> [2022-04-11 16:21:58 +0000] [14] [INFO] Booting worker with pid: 14
> [2022-04-11 16:21:58 +0000] [15] [INFO] Booting worker with pid: 15
> [2022-04-11 16:21:58 +0000] [16] [INFO] Booting worker with pid: 16
> [2022-04-11 16:21:58 +0000] [17] [INFO] Booting worker with pid: 17
> [2022-04-11 16:21:58 +0000] [18] [INFO] Booting worker with pid: 18
> [2022-04-11 16:21:59 +0000] [15] [ERROR] Exception in worker process
> Traceback (most recent call last):
>   File "/usr/local/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
>     worker.init_process()
>   File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
>     self.load_wsgi()
>   File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
>     self.wsgi = self.app.wsgi()
>   File "/usr/local/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
>     self.callable = self.load()
>   File "/usr/local/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
>     return self.load_wsgiapp()
>   File "/usr/local/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
>     return util.import_app(self.app_uri)
>   File "/usr/local/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
>     mod = importlib.import_module(module)
>   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
>     return _bootstrap._gcd_import(name[level:], package, level)
>   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
>   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
>   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
>   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
>   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
>   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
>   File "/usr/local/src/pfcon/pfcon/wsgi.py", line 9, in <module>
>     application = create_app()
>   File "/usr/local/src/pfcon/pfcon/app.py", line 14, in create_app
>     config_obj = ProdConfig()
>   File "/usr/local/src/pfcon/pfcon/config.py", line 136, in __init__
>     self.PFCON_USER = env('PFCON_USER')
>   File "/usr/local/lib/python3.8/site-packages/environs/__init__.py", line 80, in method
>     raise EnvError('Environment variable "{}" not set'.format(proxied_key or parsed_key))
> environs.EnvError: Environment variable "PFCON_USER" not set

--- a/kubernetes/remote.yaml
+++ b/kubernetes/remote.yaml
@@ -47,6 +47,10 @@ spec:
           ports:
             - containerPort: 30005
           env:
+            - name: PFCON_USER
+              value: "pfcon"
+            - name: PFCON_PASSWORD
+              value: "pfcon1234"  
             - name: SECRET_KEY
               value: "w1kxu^l=@pnsf!5piqz6!!5kdcdpo79y6jebbp+2244yjm*#+k"
             - name: COMPUTE_SERVICE_URL

